build/build.rb

```diff
@@ -89,6 +89,10 @@ def verify_run(output)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{git_checkout.commit_sha}.#{FILE_TYPE[imgfac_target]}"
     destination = DESTINATION_DIRECTORY.join(file_name)
     $log.info `mv  #{source} #{destination}`
+    $log.info "Generating image checksums"
```

Comment:
@jvlcek  Thanks Joe,...I do understand and you were very clear with original suggestions.
I wasn't asking how to implement your suggestions,...

What I'm asking is this,....there are 4 other places in the whole script that use the same format of:
cd #{destination_directory}

So being that was the format that was used in the script, I followed that example.

I can certainly make the suggested changes, and I do know how,...but if we're changing from one format for 3 new lines,...what about the existing 4 other places in the script,...should those be changes as well?